packages/suite/src/views/wallet/send/Options/BitcoinOptions/CoinControl/UtxoSelectionList/UtxoSelection/UtxoSelection.tsx

```diff
@@ -155,7 +185,10 @@ export const UtxoSelection = ({ transaction, utxo }: UtxoSelectionProps) => {
     const isChecked = isCoinControlEnabled
         ? selectedUtxos.some(selected => isSameUtxo(selected, utxo))
         : composedInputs.some(u => u.prev_hash === utxo.txid && u.prev_index === utxo.vout);
-    const isDisabled = coinjoinRegisteredUtxos.includes(utxo);
+
+    const unspendableTooltip = resolveUtxoSpendable({ utxo, coinjoinRegisteredUtxos });
```

Comment:
`resolveUtxoSpendable` function is actually a component, so let's rename it and use it as such:

Something like this:
```suggestion
    const unspendableTooltip = <UtxoUnspendableTooltip utxo={utxo} coinjoinRegisteredUtxos={coinjoinRegisteredUtxos} />
```

Reply:
solved in person